app/code/Magento/Checkout/view/frontend/web/js/product-prices-updated.js

```diff
@@ -11,19 +11,42 @@ define([
     return function (data) {
         var cacheKey = 'cart-cache-updated',
             cartData = customerData.get('cart'),
-            cartCacheUpdated = window.localStorage.getItem(cacheKey);
+            localStorageSupported = isLocalStorageSupported(),
+            cartCacheUpdated = localStorageSupported ? window.localStorage.getItem(cacheKey) : 0;
 
         customerData.getInitCustomerData().done(function () {
-            var cartUpdated = cartData().updated_at;
+            var cartUpdated = cartData().updatedAt;
+            debugger;
             if (!cartCacheUpdated || cartUpdated > cartCacheUpdated) {
                 cartCacheUpdated = cartUpdated;
             }
+
             if (data.productsUpdatedTime > cartCacheUpdated) {
                 customerData.invalidate(['cart']);
                 customerData.reload(['cart'], true);
                 cartCacheUpdated = data.productsUpdatedTime;
-                window.localStorage.setItem(cacheKey, cartCacheUpdated);
+                if (localStorageSupported) {
+                    window.localStorage.setItem(cacheKey, cartCacheUpdated);
```

Comment:
I still think we need to have there some try-catch because exception might be thrown in case if we reached localstorage limits